pkg/loader/executor.go

```diff
@@ -250,19 +294,31 @@ func (e *executor) execTableBatch(ctx context.Context, dmls []*DML) error {
 	log.Debug("merge dmls", zap.Reflect("dmls", dmls), zap.Reflect("merged", types))
 
 	if allDeletes, ok := types[DeleteDMLType]; ok {
-		if err := e.splitExecDML(ctx, allDeletes, e.bulkDelete); err != nil {
+		bulkDelete := e.bulkDelete
+		if e.destDBType == "oracle" {
+			bulkDelete = e.oracleBulkOperation
+		}
+		if err := e.splitExecDML(ctx, allDeletes, bulkDelete); err != nil {
 			return errors.Trace(err)
 		}
 	}
 
 	if allInserts, ok := types[InsertDMLType]; ok {
-		if err := e.splitExecDML(ctx, allInserts, e.bulkReplace); err != nil {
+		bulkInsert := e.bulkReplace
+		if e.destDBType == "oracle" {
+			bulkInsert = e.oracleBulkOperation
+		}
+		if err := e.splitExecDML(ctx, allInserts, bulkInsert); err != nil {
 			return errors.Trace(err)
 		}
 	}
 
 	if allUpdates, ok := types[UpdateDMLType]; ok {
-		if err := e.splitExecDML(ctx, allUpdates, e.bulkReplace); err != nil {
+		bulkUpdate := e.bulkReplace
+		if e.destDBType == "oracle" {
+			bulkUpdate = e.oracleBulkOperation
```

Comment:
Does oracle have bulk operation like MySQL? `e.oracleBulkOperation` is actually not a bulk execution.
For example:
```sql
insert into xx (aa, bb) values
(1,1),
(2,2),
(3,3);
```

Reply:
DBA will give me some statements like mysql on next version.

Reply:
ok, can we left a TODO here?
```
			// TODO: add bulkOperation for oracle
			bulkUpdate = e.oracleBulkOperation
```

Reply:
yes